<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="68310" started-at="2020-01-18T11:23:39 IST" finished-at="2020-01-18T11:24:48 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="68310" started-at="2020-01-18T11:23:39 IST" finished-at="2020-01-18T11:24:48 IST">
      <class name="com.techouts.fanniemae.testcases.RealEstateBrokerSupplierCreationTest">
        <test-method status="PASS" signature="beforeSuite()[pri:0, instance:com.techouts.fanniemae.testcases.RealEstateBrokerSupplierCreationTest@2002348]" name="beforeSuite" is-config="true" duration-ms="315" started-at="2020-01-18T11:23:39 IST" finished-at="2020-01-18T11:23:39 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method status="FAIL" signature="createRealEstateBrokerSupplier()[pri:4, instance:com.techouts.fanniemae.testcases.RealEstateBrokerSupplierCreationTest@2002348]" name="createRealEstateBrokerSupplier" duration-ms="68273" started-at="2020-01-18T11:23:39 IST" finished-at="2020-01-18T11:24:48 IST">
          <exception class="com.techouts.fanniemae.exceptions.FannieMaeBusinessException">
            <message>
              <![CDATA[throwing exception to make sure capturing testcase result to testNG]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.techouts.fanniemae.exceptions.FannieMaeBusinessException: throwing exception to make sure capturing testcase result to testNG
at com.fanniemae.automation.test.AbstractTest.performPostExceptionProcess(AbstractTest.java:236)
at com.techouts.fanniemae.testcases.RealEstateBrokerSupplierCreationTest.createRealEstateBrokerSupplier(RealEstateBrokerSupplierCreationTest.java:126)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: com.techouts.fanniemae.exceptions.FannieMaeBusinessException: unable to perform logout operation in TestCase[CreateRealEstateBrokerSupplier].
at com.fanniemae.automation.test.page.actions.HomePageActions.logout(HomePageActions.java:129)
at com.fanniemae.automation.test.AbstractTest.performPostExceptionProcess(AbstractTest.java:233)
... 29 more
]]>
            </full-stacktrace>
          </exception> <!-- com.techouts.fanniemae.exceptions.FannieMaeBusinessException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createRealEstateBrokerSupplier -->
        <test-method status="PASS" signature="afterSuite()[pri:0, instance:com.techouts.fanniemae.testcases.RealEstateBrokerSupplierCreationTest@2002348]" name="afterSuite" is-config="true" duration-ms="4635" started-at="2020-01-18T11:24:48 IST" finished-at="2020-01-18T11:24:52 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- com.techouts.fanniemae.testcases.RealEstateBrokerSupplierCreationTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
